pyntc/devices/ios_device.py

```diff
@@ -227,21 +254,93 @@ def close(self):
             self.native.disconnect()
             self._connected = False
 
-    def config(self, command):
-        self._enter_config()
-        self._send_command(command)
-        self.native.exit_config_mode()
+    def config(self, command, **netmiko_args):
+        """
+        Send config commands to device.
+
+        By default, entering and exiting config mode is handled automatically.
+        To disable entering and exiting config mode, pass `enter_config_mode` and `exit_config_mode` in ``**netmiko_args``.
+        This supports all arguments supported by Netmiko's `send_command_set` method using ``netmiko_args``.
+        This will send all commands in ``command`` regardless of failures, and only raise an Error after sending all commands.
+
+        Args:
+            command (str|list): The command or commands to send to the device.
```

Comment:
Seems that config should be able to handle a list of commands but the implementation does not indicate that.

Reply:
netmiko's `send_config_set` method supports both a single command as a string, out a list of commands to send. By switching to use the `send_config_set` method, we would have to do data validation, and I don't see what we would gain by enforcing a single command (open to hear the case for enforcement though)

Reply:
I agree with all that you said. The only gain would be "do not repeat yourself" type of gain, but in this case it is small and I can really see the case for the different errors that the two commands are throwing.